public/manifest/production.json

```diff
@@ -18,5 +18,44 @@
       "display_name": "Ethiopia",
       "isd_code": "251"
     }
+  ],
+  "v2": [
+    {
+      "country_code": "IN",
+      "display_name": "India",
+      "isd_code": "91",
+      "deployments": [
+        {
+          "display_name": "IHCI",
+          "endpoint": "https://api.simple.org/api/"
+        },
+        {
+          "display_name": "Kerala",
+          "endpoint": "https://api.kerala.simple.org/api/"
```

Comment:
This doesn't exist yet, shall we not include it in the prod manifest until it is?

Reply:
Removed